src/nvim/screen.c

```diff
@@ -5672,19 +5704,19 @@ void screenalloc(void)
   resizing = false;
 }
 
-/// Clear tab_page_click_defs table
+/// Clear status line, window bar or tab page line click definition table
 ///
 /// @param[out]  tpcd  Table to clear.
 /// @param[in]  tpcd_size  Size of the table.
-void clear_tab_page_click_defs(StlClickDefinition *const tpcd, const long tpcd_size)
+void clear_click_defs(StlClickDefinition *const click_defs, const long click_defs_size)
```

Comment:
common prefixes are very helpful

```suggestion
void stl_clear_click_defs(StlClickDefinition *const click_defs, const long click_defs_size)
```

Reply:
done